src/main/kotlin/no/nav/syfo/melding/domain/MeldingTilBehandler.kt

```diff
@@ -223,14 +225,15 @@ fun MeldingTilBehandler.toJournalpostRequest(pdf: ByteArray) =
 fun MeldingTilBehandler.createTittel(): String {
     return when (this.type) {
         MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER, MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING, MeldingType.HENVENDELSE_MELDING_FRA_NAV -> MeldingTittel.DIALOGMELDING_DEFAULT.value
+        MeldingType.HENVENDELSE_MELDING_TIL_NAV -> MeldingTittel.DIALOGMELDING_TIL_NAV.value
         MeldingType.FORESPORSEL_PASIENT_PAMINNELSE -> MeldingTittel.DIALOGMELDING_PAMINNELSE.value
         MeldingType.HENVENDELSE_RETUR_LEGEERKLARING -> MeldingTittel.DIALOGMELDING_RETUR.value
     }
 }
 
 fun MeldingTilBehandler.createOverstyrInnsynsregler(): String? {
     return when (this.type) {
-        MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER, MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING, MeldingType.HENVENDELSE_RETUR_LEGEERKLARING, MeldingType.HENVENDELSE_MELDING_FRA_NAV -> OverstyrInnsynsregler.VISES_MASKINELT_GODKJENT.value
+        MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER, MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING, MeldingType.HENVENDELSE_RETUR_LEGEERKLARING, MeldingType.HENVENDELSE_MELDING_FRA_NAV, MeldingType.HENVENDELSE_MELDING_TIL_NAV -> OverstyrInnsynsregler.VISES_MASKINELT_GODKJENT.value
```

Comment:
Er det avklart at disse meldingene skal vises for bruker på mine saker?

Reply:
Nei, det er sant. Det må vi sjekke.

Reply:
Disse reglene gjelder kanskje bare for utgående meldinger? 

when(type) krever at vi håndterer alle typer, så det virker det riktigst å mappe typen for innkommende melding til null.